### ISSUE TITLE
[BUG] : <Can't download some songs>

### ISSUE THREAD

**Issue by adityaharyadi** [Shabinder/SpotiFlyer#971]
**Describe the bug:**
Some of the songs in the playlist that I attached can't be downloaded 

**Media Links Used:** 
https://open.spotify.com/playlist/7kRXaUMd7tHE7Vs5GCGf29?si=I-3guVp3RyuLUxp_iyl-oA&utm_source=copy-link

**Expected behavior**
I hope that the song I want to download can be downloaded 

**Screenshots:**
![Screenshot_20220125-183930](https://user-images.githubusercontent.com/98391624/150961661-b582c69f-8948-4a23-af46-f98afca5c004.png)

**StackTrace:**
```
Paste Stacktrace here if available
java.lang.Exception: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
	at com.shabinder.spotiflyer.service.ForegroundService$enqueueDownload$$inlined$collect$1.emit(Collect.kt:9)
	at com.shabinder.common.core_components.file_manager.FileManagerKt$downloadFile$4.invokeSuspend(FileManager.kt:5)
	at com.shabinder.common.core_components.file_manager.FileManagerKt$downloadFile$4.invoke(Unknown Source:11)
	at com.shabinder.common.core_components.file_manager.FileManagerKt$downloadFile$4.invoke(Unknown Source:6)
	at kotlinx.coroutines.flow.FlowKt__ErrorsKt$catch$$inlined$unsafeFlow$1.collect(SafeCollector.common.kt:6)
	at kotlinx.coroutines.flow.FlowKt__ErrorsKt$catch$$inlined$unsafeFlow$1$1.invokeSuspend(Unknown Source:12)
	at u.v.j.a.a.resumeWith(ContinuationImpl.kt:3)
	at io.ktor.util.pipeline.SuspendFunctionGun.resumeRootWith(SuspendFunctionGun.kt:12)
	at io.ktor.util.pipeline.SuspendFunctionGun.access$resumeRootWith(SuspendFunctionGun.kt:1)
	at io.ktor.util.pipeline.SuspendFunctionGun$continuation$1.resumeWith(SuspendFunctionGun.kt:2)
	at u.v.j.a.a.resumeWith(ContinuationImpl.kt:9)
	at io.ktor.util.pipeline.SuspendFunctionGun.resumeRootWith(SuspendFunctionGun.kt:12)
	at io.ktor.util.pipeline.SuspendFunctionGun.access$resumeRootWith(SuspendFunctionGun.kt:1)
	at io.ktor.util.pipeline.SuspendFunctionGun$continuation$1.resumeWith(SuspendFunctionGun.kt:2)
	at u.v.j.a.a.resumeWith(ContinuationImpl.kt:9)
	at io.ktor.util.pipeline.SuspendFunctionGun.resumeRootWith(SuspendFunctionGun.kt:12)
	at io.ktor.util.pipeline.SuspendFunctionGun.access$resumeRootWith(SuspendFunctionGun.kt:1)
	at io.ktor.util.pipeline.SuspendFunctionGun$continuation$1.resumeWith(SuspendFunctionGun.kt:2)
	at u.v.j.a.a.resumeWith(ContinuationImpl.kt:9)
	at io.ktor.util.pipeline.SuspendFunctionGun.resumeRootWith(SuspendFunctionGun.kt:12)
	at io.ktor.util.pipeline.SuspendFunctionGun.access$resumeRootWith(SuspendFunctionGun.kt:1)
	at io.ktor.util.pipeline.SuspendFunctionGun$continuation$1.resumeWith(SuspendFunctionGun.kt:2)
	at u.v.j.a.a.resumeWith(ContinuationImpl.kt:9)
	at io.ktor.util.pipeline.SuspendFunctionGun.resumeRootWith(SuspendFunctionGun.kt:12)
	at io.ktor.util.pipeline.SuspendFunctionGun.access$resumeRootWith(SuspendFunctionGun.kt:1)
	at io.ktor.util.pipeline.SuspendFunctionGun$continuation$1.resumeWith(SuspendFunctionGun.kt:2)
	at u.v.j.a.a.resumeWith(ContinuationImpl.kt:9)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:17)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:1)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:4)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:4)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:1)


**Device Info (please complete the following information):**
 - Device:   Samsung A51
 - OS:   Android 10
 - Version:   I don't know what version you mean
 - Country: Indonesia

**Additional context:**
I don't have any additional context.

**Comment by Shabinder** [comment]
Fixed in **v3.6.0** : https://github.com/Shabinder/SpotiFlyer/releases/tag/v3.6.0

**Comment by BlazeHell** [comment]
Did your problem get fixed because I'm also have the same issue
